_content/excerpts/itineraries-sihala-monk-saralanka_kirichenko-alexey.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "The Itineraries of “Sīhaḷa Monk” Sāralaṅkā"
+authors:
+  - "Alexey Kirichenko"
+external_url: "https://muse.jhu.edu/pub/5/oa_edited_volume/chapter/2379328/pdf"
+source_url: "https://manifold.uhpress.hawaii.edu/read/buddhist-and-islamic-orders-in-southern-asia-comparative-perspectives/section/d58efeec-6889-4098-a9a4-92c14e44c46d"
+drive_links:
+  - "https://drive.google.com/file/d/1D3v6tfZAdCfYWN6QhqOo4ZSWtX4UbGBk/view?usp=sharing"
+course: theravada-roots
+tags:
```

Comment:
Yeah, this is a hard one to tag. It's pan regional, also highlighting the Burmese reactions, so focusing on Sri Lanka seems inappropriate here. It's also a descriptive, not prescriptive, study, so vinaya-studies is not appropriate.  Let's go with `monastic`, perhaps? As this is ultimately about community dynamics... 🤔  Perhaps we can even add `nationalism` as that seemed to be a large motivating factor here.